cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -370,6 +370,10 @@ public static final class AppMetadataStore {
     public static final String APPLICATION_FIELD = "application";
     public static final String VERSION_FIELD = "version";
     public static final String APPLICATION_DATA_FIELD = "application_data";
+    public static final String CREATION_TIME_FIELD = "created";
+    public static final String CHANGE_SUMMARY_FIELD = "change_summary";
+    public static final String AUTHOR_FIELD = "author";
```

Comment:
So we updated this field from `owner` to `author`? And decided to add `change_summary`? We'd better document them in the design doc to prevent inconsistency

Reply:
yes updated the doc, thanks!